checkpoints/check_wifi.py

```diff
@@ -33,7 +33,7 @@ def check_wifi_is_configured():
 
 def check_wifi_is_connected():
     """Check wlan0 has an IP address."""
-    output = subprocess.check_output(['ifconfig', 'wlan0']).decode('utf-8')
+    output = subprocess.check_output(['sudo', 'ifconfig', 'wlan0']).decode('utf-8')
```

Comment:
hmm, on raspian this is not needed and we'd like to make it not more complicated to run this for users there. Maybe we can find an alternative way to check if wifi is up that is more portable and won't require sudo.
Do you think it would be useful to bring up some of the interop issue on hassbian, maybe they can make it more compatible too.